tests/kernel/boot_ltp.pm

```diff
@@ -58,6 +58,11 @@ sub run {
 
     log_versions;
 
+    # Enable nfs service for ltp test
+    if (is_sle_micro) {
+	    assert_script_run('systemctl restart nfs-server');
```

Comment:
It would be better to enable service in `install_ltp` to have image in final state.

Reply:
Moreover, we are skipping whole ` setup_network` function in install_ltp`. I think we should try it to run first to see what are the issues and adapt it for micro.

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/a1b8ef6dba68a1088dfba2c3d3330612fa91a1b5/tests/kernel/install_ltp.pm#L375

Reply:
Following fail happen if you directly enable setup_network. So i do some update on this PR and also update VR in description.

https://openqa.suse.de/tests/14794807#step/install_ltp/319
touch: cannot touch '/var/lib/dhcp/db/dhcpd.leases': No such file or directory

https://openqa.suse.de/tests/14794825#step/install_ltp/319
sed: couldn't open temporary file /usr/etc/sedn4Ht0q: Read-only file system